.ci/docker/common/install_python.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This file is supposed to be like install_conda, but with a lot of things taken out since I'm not sure if they're needed. As a side note, its probably better to reduce conditionals in these scripts to allow for layer reuse?